addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/channel-groups.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Channel Groups -->
+
+	<channel-group-type id="zone1Controls">
```

Comment:
@paulianttila I only came across this now (and had missed it when reviewing the PR): Why did you actually create 3 group-types for the 3 zones? The idea of the groups is that you only define them once and then reuse the same type for the 3 zones. Would it be possible to refactor that binding or is there any reason against doing so?

Reply:
@kaikreuzer, reason was that different zones have different amount of features. Currently all zones have the same amount of features, but it's just matter of time when new zone based features will be introduced. OH1 Onkyo binding supported all features of the receivers, so OH2 channel support is still quite limited.

Reply:
Ok, thanks. This is actually a bit weird that the different zones don't have the same features, but ok.
Note that the original idea of channel-groups was to group and reuse sets of features and not to merely structure the flat list of available channels in some hierarchy. But maybe this original intent needs to be re-discussed, so let's leave it here as it is for now.